modules/dns/dns.rb

```diff
@@ -1,20 +1 @@
 require 'dns/dns_plugin'
-
-module Proxy::Dns
```

Comment:
The removal of these is going to cause a compatibility issue for the DNS plugins between 1.10 and 1.11.  Do they need to be moved into dns_common?